<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDetailInfoHorse</class>
 <widget class="QWidget" name="FormDetailInfoHorse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="mainName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MathJax_Fraktur</family>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SKachKi</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Кличка:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sexlabel">
         <property name="text">
          <string>Пол:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>Возраст:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ownerLabel">
         <property name="text">
          <string>Владелец:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="availableLabel">
         <property name="text">
          <string>Учавствует сейчас:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="nameHorse">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sex">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="age">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="owner">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="available">
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="name">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="date">
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time">
       <property name="text">
        <string>Время</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="place">
       <property name="text">
        <string>Место</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stat">
       <property name="text">
        <string>Статус</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="moreInfoCol">
       <property name="text">
        <string>Подробности</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1037</width>
        <height>313</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
